data/extract_hubert_centroids.py

```
@@ -0,0 +1,254 @@
+    def __getitem__(self, idx):
+        audio_path = self.audio_files[idx]
+        
+        try:
+            # Load audio with librosa (more robust)
+            waveform, sr = librosa.load(audio_path, sr=self.sample_rate, mono=True)
+            
+            # Convert to tensor
+            waveform = torch.from_numpy(waveform).float()
+            
+            return waveform
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)